client/packages/invoices/src/InboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -51,15 +57,21 @@ const getStatusOptions = (
   ];
 
   if (currentStatus === InvoiceNodeStatus.Shipped) {
-    // When the status is shipped, delivered and verified are available to
+    // When the status is shipped, delivered, received and verified are available to
     // select.
     options[3].isDisabled = false;
     options[4].isDisabled = false;
+    options[5].isDisabled = false;
   }
 
-  // When the status is delivered, only verified is available to select.
+  // When the status is Delivered, only verified & Delivered re available to select.
   if (currentStatus === InvoiceNodeStatus.Delivered) {
     options[4].isDisabled = false;
+    options[5].isDisabled = false;
```

Comment:
Would be nice to refactor to not rely on the indexes here, as it's really messy to update and to verify if it's correct